pom.xml

```diff
@@ -294,6 +295,30 @@
         </dependency>
         <dependency>
             <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-compatibility-server</artifactId>
```

Comment:
Good catch.  Found one unused import still referencing them, but otherwise I'd updated uses but left the dependencies.  Will remove them in a new pull request.